server/views/applications/_table.njk

```diff
@@ -0,0 +1,31 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% macro applicationsTable(title, rows) %}
+    {% if rows %}
+        {{
+      govukTable({
+        caption: title,
+        captionClasses: "govuk-table__caption--m",
+        firstCellIsHeader: true,
+        head: [
+          {
+            text: "Service user"
+          },
+          {
+            text: "CRN"
+          },
+          {
+            text: "Tier"
+          },
+          {
+            text: "Arrival date"
+          },
+            {
+            text: "Status"
+          }
+        ],
+        rows: rows
+      })
```

Comment:
Minor point, but do you think you could fix the alignment of this object? The alignment is a bit off

Reply:
of course